.github/issue_template.md

```diff
@@ -1,7 +1,17 @@
-Note: This is just a template, so feel free to use/remove the unnecessary things
+_Note:_ this is a template, please remove the parts that are not
+applicable (these initial notes, and the "Bug" section for a Feature request
+and vice-versa).
 
+**Note:** to report a security vulnerability, see
```

Comment:
Maybe we could explain that by security vulnerability we mean any bug that might have a security impact (eg. memory corruption errors), they don't have to have an PoC exploit taking over the machine ready to go.